src/sv-pipeline/scripts/aggregate_picard.py

```diff
@@ -0,0 +1,276 @@
+                                       'WORST_CXT_6', 'WORST_CXT_QSCORE_6', 
+                                       'WORST_PRE_CXT_6', 'WORST_PRE_CXT_QSCORE_6', 'WORST_POST_CXT_6',
+                                       'WORST_POST_CXT_QSCORE_6'], axis = 1)
+    sadfseven = pd.DataFrame(table.loc[7]).T.reset_index(drop = True)
+    sadf7 = sadfseven.add_suffix("_7").drop(['SAMPLE_ALIAS_7', 'LIBRARY_7', 
+                                         'WORST_CXT_7', 'WORST_CXT_QSCORE_7', 
+                                         'WORST_PRE_CXT_7', 'WORST_PRE_CXT_QSCORE_7', 'WORST_POST_CXT_7',
+                                       'WORST_POST_CXT_QSCORE_7'], axis = 1)
+    sadfeight = pd.DataFrame(table.loc[8]).T.reset_index(drop = True)
+    sadf8 = sadfeight.add_suffix("_8").drop(['SAMPLE_ALIAS_8', 'LIBRARY_8', 'WORST_CXT_8', 
+                                         'WORST_CXT_QSCORE_8', 'WORST_PRE_CXT_8', 
+                                         'WORST_PRE_CXT_QSCORE_8', 'WORST_POST_CXT_8',
+                                       'WORST_POST_CXT_QSCORE_8'], axis = 1)
+    sadfnine = pd.DataFrame(table.loc[9]).T.reset_index(drop = True)
+    sadf9 = sadfnine.add_suffix("_9").drop(['SAMPLE_ALIAS_9', 'LIBRARY_9', 
+                                        'WORST_CXT_9', 'WORST_CXT_QSCORE_9', 
+                                        'WORST_PRE_CXT_9', 'WORST_PRE_CXT_QSCORE_9', 'WORST_POST_CXT_9',
+                                       'WORST_POST_CXT_QSCORE_9'], axis = 1)
+    sadften = pd.DataFrame(table.loc[10]).T.reset_index(drop = True)
+    sadf10 = sadften.add_suffix("_10").drop(['SAMPLE_ALIAS_10', 'LIBRARY_10', 
+                                         'WORST_CXT_10', 'WORST_CXT_QSCORE_10', 
+                                         'WORST_PRE_CXT_10', 'WORST_PRE_CXT_QSCORE_10', 'WORST_POST_CXT_10',
+                                       'WORST_POST_CXT_QSCORE_10'], axis = 1)
+    sadfeleven = pd.DataFrame(table.loc[11]).T.reset_index(drop = True)
+    sadf11 = sadfeleven.add_suffix("_11").drop(['SAMPLE_ALIAS_11', 'LIBRARY_11', 
+                                            'WORST_CXT_11', 'WORST_CXT_QSCORE_11', 
+                                            'WORST_PRE_CXT_11', 'WORST_PRE_CXT_QSCORE_11', 'WORST_POST_CXT_11',
+                                       'WORST_POST_CXT_QSCORE_11'], axis = 1)
+    sadftwelve = pd.DataFrame(table.loc[12]).T.reset_index(drop = True)
+    sadf12 = sadftwelve.add_suffix("_12").drop(['SAMPLE_ALIAS_12', 'LIBRARY_12', 
+                                            'WORST_CXT_12', 'WORST_CXT_QSCORE_12', 'WORST_PRE_CXT_12', 
+                                            'WORST_PRE_CXT_QSCORE_12', 'WORST_POST_CXT_12',
+                                       'WORST_POST_CXT_QSCORE_12'], axis = 1)
+    SAMresult = pd.concat([sadf1,sadf2, sadf3, sadf4, sadf5, sadf6, sadf7, sadf8, sadf9, sadf10, sadf11, sadf12 ], axis = 1)
+    return SAMresult
+
+def windows(table, window):
+    if len(table.columns) == 2:
```

Comment:
I think this is fine to use in the script, but in general having the output of the function be conditioned on the hardcoded variable like this is a sign that the code should be refactored. 

One way to refactor it is to rewrite this code in an object-oriented manner. Again, I don't think it is necessary here but for code that is more than few hundred lines long it's a good idea, and it will provide more readability and robustness.